app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStep.js

```diff
@@ -44,9 +45,9 @@ class PlanWizardVMStep extends React.Component {
     const { csvParseErrorAction } = this.props;
     csvParseErrorAction(errMsg);
   };
-  validateVms = () => {
+  validateVms = (vms = [], planId = null, meta = {}) => {
```

Comment:
I think it doesn't matter since you've provided defaults, but since you're changing the function signature, should we also update [this line?](https://github.com/ManageIQ/manageiq-v2v/blob/efe628b8d5aea578a0cb74c7da49e422925e0edc/app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStep.js#L29)

Reply:
We can if you think it would be best, but that's why I provided defaults. No arguments = discovery of VMs with no prior information. I'm ok with either way. Would that be your preference?

Reply:
I was just sitting here and pondering this.  It's not a big deal, just a functionality vs. convention thing lol

When it comes to assigning default values to function parameters, I think of it as a way to guard against inadvertently passing `undefined`.  On the other hand, if I want to have truly optional arguments, I like to use an object.  This allows us to avoid having to pass `null`

```js
this.validateVms([], null, { combineRequests: !!shouldPrefillForEditing })
```

Anyway, either way works, and since we've never really set a convention one way or another, I should not be so picky about it.  I think we're good to go :)